util/find-doc-nits

```diff
@@ -22,6 +22,10 @@ use OpenSSL::Util::Pod;
 # Set to 1 for debug output
 my $debug = 0;
 
+# Where to find openssl command
+my $BLDTOP  = $ENV{BLDTOP} || ".";
+my $openssl = "$BLDTOP/util/opensslwrap.sh";
```

Comment:
Er, do we actually copy `util/opensslwrap.sh` to the build tree?

Reply:
Yes it gets copied; see https://github.com/openssl/openssl/blob/master/Configurations/unix-Makefile.tmpl#L1086  At least on Unix.

Reply:
Ah, I had forgotten...  well, since if only works on Unix-like shells, that's ok.